src/accessibility/contact-me/index.md.njk

```diff
@@ -0,0 +1,18 @@
+---
+layout: layout-pane.njk
+title: Contact me
+description: Contact Gaynor
+show_subnav: true
+---
+
+##Contact me
```

Comment:
two heading in the page - doesn't need the ## Contact me here

Reply:
Removed second title